res/skins/Deere/deck_controls_row.xml

```diff
@@ -52,16 +52,28 @@
           <!-- Workaround for layout spacing -->
           <WidgetGroup><Size>0min,2f</Size></WidgetGroup>
 
-          <Template src="skin:left_right_display_2state_button.xml">
-            <SetVariable name="TooltipId">play_cue_set</SetVariable>
-            <SetVariable name="ObjectName">PlayToggle</SetVariable>
-            <SetVariable name="MinimumSize"><Variable name="HorizontalStretchButtonMinimumSize"/></SetVariable>
-            <SetVariable name="MaximumSize"><Variable name="HorizontalStretchButtonMaximumSize"/></SetVariable>
-            <SetVariable name="SizePolicy"><Variable name="HorizontalStretchButtonSizePolicy"/></SetVariable>
-            <SetVariable name="left_connection_control"><Variable name="group"/>,play</SetVariable>
-            <SetVariable name="right_connection_control"><Variable name="group"/>,cue_set</SetVariable>
-            <SetVariable name="display_connection_control"><Variable name="group"/>,play_indicator</SetVariable>
-          </Template>
+          <WidgetGroup><!-- Play button + play_indicator -->
+            <Layout>stacked</Layout>
```

Comment:
Hmm, this is an interesting hack but I suspect there is a bug in WPushButton or in the engine's handling of `play` or `play_handling` at the root of this...

Reply:
yeah, but maybe it's simply indicator confusion for buttons with two connections affecting the displayValue. now the Play button has only one connnection.